ctapipe/reco/stereo_combination.py

```diff
@@ -0,0 +1,290 @@
+from abc import abstractmethod
+
+import astropy.units as u
+import numpy as np
+from astropy.table import Table
+
+from ctapipe.core import Component, Container
+from ctapipe.core.traits import Bool, CaselessStrEnum, Unicode
+
+from ..containers import (
+    ArrayEventContainer,
+    ParticleClassificationContainer,
+    ReconstructedEnergyContainer,
+)
+from .utils import add_defaults_and_meta
+
+_containers = {
+    "energy": ReconstructedEnergyContainer,
+    "classification": ParticleClassificationContainer,
+}
+
+__all__ = [
+    "StereoCombiner",
+    "StereoMeanCombiner",
+]
+
+
+def _calculate_ufunc_of_telescope_values(tel_data, n_array_events, indices, ufunc):
+    combined_values = np.zeros(n_array_events)
+    ufunc.at(combined_values, indices, tel_data)
```

Comment:
This works only for additive ufuncs, right? Not for e.g. multiplication. We currently only use it for `np.add` though, but that might become an issue

Reply:
maybe better hard code `add` for now and call it `grouped_add`?

Reply:
yes